src/plugins/blocksmith/components/image-block-modal/CustomizeOptions.jsx

```diff
@@ -51,104 +58,148 @@ const CustomizeOptions = ({
   url,
 }) => (
   <div className="modal-body-blocksmith">
-    <ImageBlockPreview className="my-4" block={block} />
-    {hasCaption && (block.has('caption') || selectedImage.has('description')) && (
-      <p>Caption: {block.get('caption') || selectedImage.get('description')}</p>
-    )}
-    <FormGroup>
-      <ImageAssetPicker
-        multiAssetErrorMessage="Invalid Action: Trying to assign multiple Gallery Block Poster images."
-        isImageSelected={!!selectedImage}
-        isModalOpen={isAssetPickerModalOpen}
-        isDisabled={false}
-        label="Select an image"
-        node={node}
-        onClearImage={handleClearImage}
-        onSelectImage={handleSelectImage}
-        onToggleAssetPickerModal={handleToggleAssetPickerModal}
-      />
-    </FormGroup>
-    <div style={{ maxWidth: '350px', margin: '0 auto' }}>
+      <ImageBlockPreview className="my-4" block={block} />
```

Comment:
@jalopez13 - one final thing: Indentation is messed up here, 6 spaces instead of 4, please fix it.

Reply:
@arunkarnati 

Fixed space indentation.